tests/parametric/conftest.py

```diff
@@ -713,6 +713,8 @@ def wait_for_rc_apply_state(
             else:
                 # Look for the given apply state in the requests.
                 for req in rc_reqs:
+                    if req["body"]["client"]["state"].get("config_states") is None:
+                        continue
```

Comment:
Golang omits `config_states` if it's empty. This check here avoids a `KeyError`

Reply:
Given this wasn't here before, is it OK to be omitting it if it's empty given other tracers seem to always include it?

Reply:
From the RC spec: `Default: Not set OR Empty list`
![image](https://github.com/DataDog/system-tests/assets/38987709/bb1e746d-d178-4d80-8c5b-93088133479b)